sitewit/models.py

```diff
@@ -57,7 +57,7 @@ def create(cls, user, url, site_id=None, user_token=None):
         Raises:
             demands.HTTPServiceError: if any error happened on HTTP level.
         """
-        email = cls._get_email(user.id)
+        email = user.preferences.get('wl_email', user.email)
```

Comment:
Maybe `user.preferences.get('wl_email', user.email)` should be wrapped in a `get_email` method so it can be easily overridden?  This is pretty tightly coupled to Yola's user preferences implementation.

Reply:
yes. new pr open: https://github.com/yola/sitewit/pull/59/files
closing this one